modules/afsocket/afunix-source.c

```diff
@@ -86,6 +87,14 @@ afunix_sd_init(LogPipe *s)
 {
   AFUnixSourceDriver *self = (AFUnixSourceDriver *) s;
 
+  if (!is_file_socket(self->filename))
```

Comment:
File socket checking before `afsocket_sd_init_method` is must because `afsocket_sd_init_method` will be false if file is not any type of socket and syslog-ng is writting out only the `Address already in use` ... error message.